packages/contracts/src/core/dao/DAO.sol

```diff
@@ -168,9 +168,9 @@ contract DAO is
 
     /// @inheritdoc IDAO
     function execute(
-        bytes32 callId,
+        bytes32 _callId,
```

Comment:
Unless there is a clearly misleading name, I would try to avoid changing the contracts code at this point. In order to speed up any follow-up post-audit review for actual changes

Reply:
This would mean that we have to also use the inconsistent notation in the examples in the developer portal, where we show-case how the functions are supposed to be used.
IMO, we would need to have a cleaning commit that fixes the NatSpec / typos and other small inconsistencies.

However, I can change it back

Reply:
Reverted in b27c7c596298fd51920df929b41da4f86b453b68